### DISCUSSION TITLE
Unconfirmed UTXO that doesn't appear in mempool.

### DISCUSSION THREAD

**Discussion by scottshell** [Ride-The-Lightning/RTL#1271] [Troubleshooting Questions] [answered]
So I had a number of dust UTXOs that were sitting in an unconfirmed state that I tried to sweep into a single transaction and via llncli command line and now I have a single UTXO that is also not being broadcast into the mempool. It does show up as an unconfirmed transaction in my own node but isn't spendable since it is not confirming due to it not being in the mempool.

Is there a means to push this transaction? Also, I don't know what fee has been set and was wondering if there was a means to determine what that is before it shows up in the mempool?

**Comment by saubyk** [comment, marked as answer]
If the unconfirmed transactions are less than two weeks old, it should still be in the mempool, you can try to bump up the unconfirmed UTXO via CPFP
https://docs.lightning.engineering/lightning-network-tools/lnd/unconfirmed-bitcoin-transactions

make sure to use a high enough fee rate (which should be more than the highest rate you see on mempool), so that the average fee rate is good enough for the miners to prioritize it.

**Reply by scottshell** [reply]
The latest transaction is less than a day old but the dust transactions in some cases were older.  Is it still possible that doing a CPFP will work in this case and is there any way to see what fee rate was set on the original transaction so that I can figure out how much of an increase I need to apply to get it in the mempool?

**Reply by saubyk** [reply]
You should be able to find that from a block explorer like mempool.space. You need to search with the transaction id of your unconfirmed transaction

**Reply by scottshell** [reply]
Unfortunately when I search for the transaction ID it isn't found in the mempool (i.e. @ mempool.space) it says it is waiting for the transaction to appear in the mempool.

**Reply by saubyk** [reply]
If it's not in the mempool, it's likely not been broadcast from your node, which might be because you used too low a fee rate. It might be best to restart your node once. If that doesn't change the state, wait for a bit (2 weeks) before your dust UTXOs are available again.
